### PR TITLE
Easy 916

### PR DESCRIPTION
**Pull request by ekoi**
Ik heb wijzigingen in DDM.scala en AddEmdToDatasetIngestDirSpec. Graag de code reviewen en testen uitvoeren.

### CODE REVIEW

File: src/main/scala/nl/knaw/dans/easy/ps/DDM.scala [hunk 1]
```diff
@@ -54,7 +54,7 @@ object DDM {
   def isPartOfComposedContributor(key: MdKey) = composedContributorFields.keySet.contains(key)
 
   def datasetsToXml(datasets: Datasets): Observable[String] =
-    Observable.from(datasets.map(_._2)).map(DDM.datasetToXml).map(_.toString)
+    Observable.from(datasets.map(_._2)).map(DDM.datasetToXml).map(_.toString).doOnNext(ddm => println(ddm))
```
Comment (janvanmansum) [inline src/main/scala/nl/knaw/dans/easy/ps/DDM.scala]: Why `println(ddm)`?

File: src/main/scala/nl/knaw/dans/easy/ps/DDM.scala [hunk 2]
```diff
@@ -113,12 +113,35 @@ object DDM {
 
   def createComposedCreator(dictionary: Dictionary, authorFields: Iterable[(MdKey, String)]) = {
     <dcx-dai:creatorDetails>
-      <dcx-dai:author>
-        { authorFields.map(composedEntry(dictionary)) }
-      </dcx-dai:author>
+      {
+      if (isOrganization(authorFields))
+        <dcx-dai:organization>
+          <dcx-dai:name xml:lang="zh-CN">
```
Comment (janvanmansum) [inline src/main/scala/nl/knaw/dans/easy/ps/DDM.scala]: Why `xml:lang="zh-CN"`?

File: src/main/scala/nl/knaw/dans/easy/ps/DDM.scala [hunk 3]
```diff
@@ -113,12 +113,35 @@ object DDM {
 
   def createComposedCreator(dictionary: Dictionary, authorFields: Iterable[(MdKey, String)]) = {
     <dcx-dai:creatorDetails>
-      <dcx-dai:author>
-        { authorFields.map(composedEntry(dictionary)) }
-      </dcx-dai:author>
+      {
```
Comment (janvanmansum) [inline src/main/scala/nl/knaw/dans/easy/ps/DDM.scala:116]: What happens if you mix "person" creators and "organization-only" creators? Maybe add a unit-test to show that that is handled correctly?

**Comment by janvanmansum** [comment]
@ekoi please address the issues raised in my comments.

File: src/main/scala/nl/knaw/dans/easy/ps/DDM.scala (hunk 1 above)
Comment (ekoi) [inline src/main/scala/nl/knaw/dans/easy/ps/DDM.scala]: I have to remove it.

File: src/main/scala/nl/knaw/dans/easy/ps/DDM.scala (hunk 2 above)
Comment (ekoi) [inline src/main/scala/nl/knaw/dans/easy/ps/DDM.scala]: Copy from https://drivenbydata.atlassian.net/browse/EASY-916

Note:
Dit is nog niet opgelost: <dcx-dai:name xml:lang="zh-CN"> 
zh-CN is hard gecodeerd. Ik weet 't niet van waar de waarde moet ik vullen. In de DDM.scala staat overaal hard gecodeerd xml:lang="en". De zh-CN komt uit de min.xml

File: src/main/scala/nl/knaw/dans/easy/ps/DDM.scala (hunk 3 above)
Comment (ekoi) [inline src/main/scala/nl/knaw/dans/easy/ps/DDM.scala:116]: Mixed:
dcx-dai:creatorDetails
            dcx-dai:organization
                <dcx-dai:name xml:lang="zh-CN">DC.LPEeo/dcx-dai:name
            /dcx-dai:organization

```
        <dcx-dai:author>
    <dcx-dai:titles xml:lang="en-IE">dr.</dcx-dai:titles>
            <dcx-dai:initials>A</dcx-dai:initials>
            <dcx-dai:insertions>v.d</dcx-dai:insertions>
            <dcx-dai:surname>Coen</dcx-dai:surname>
            <dcx-dai:DAI>93313935x</dcx-dai:DAI> 
            <dcx-dai:organization>
                <dcx-dai:name xml:lang="zh-CN">A.C</dcx-dai:name>
            </dcx-dai:organization>
        </dcx-dai:author>
    </dcx-dai:creatorDetails>
```

Test:
it should "fail if person creators and organization-only creators are mixed" in {

```
AddEmdToDatasetIngestDir("1", {enoughEmd +=
  ("DCX_CREATOR_ORGANIZATION" -> List("dans"))
}) checkPreconditionsShould be(a[Success[_]])
```

  }

File: src/test/scala/nl/knaw/dans/easy/ps/AddEmdToDatasetIngestDirSpec.scala [hunk 4]
```diff
@@ -47,6 +47,13 @@ class AddEmdToDatasetIngestDirSpec extends UnitSpec {
     }) checkPreconditionsShould be(a[Success[_]])
   }
 
+it should "fail if person creators and organization-only creators are mixed" in {
```
Comment (janvanmansum) [inline src/test/scala/nl/knaw/dans/easy/ps/AddEmdToDatasetIngestDirSpec.scala:50]: It should _not_ fail. That was my point. Use the Web-UI as a reference of what is permissible.

File: src/test/scala/nl/knaw/dans/easy/ps/AddEmdToDatasetIngestDirSpec.scala [hunk 5]
```diff
@@ -38,7 +38,7 @@ class AddEmdToDatasetIngestDirSpec extends UnitSpec {
     action checkPreconditionsShould be(a[Success[_]])
   }
 
-  ignore should "succeeed with just an organisation as creator" in {
+  it should "succeeed with just an organisation as creator" in {
```
Comment (janvanmansum) [inline src/test/scala/nl/knaw/dans/easy/ps/AddEmdToDatasetIngestDirSpec.scala:41]: typo: "succeeeeeed"

**Pull request merged**